parts/k8s/kubernetesmastervars.t

```diff
@@ -86,10 +86,18 @@
     "masterOffset": "[parameters('masterOffset')]",
     {{ end }}
 {{end}}
+{{if not IsAzureStackCloud}}
```

Comment:
@honcao Can we flip this? I prefer using it like this
```suggestion
{{if IsAzureStackCloud}}
```

Reply:
@tariq1890 , thanks for the comments, I updated the code.